config/ddpConfig.js.ctmpl

```diff
@@ -27,10 +27,10 @@ var DDP_ENV = {
      {{end}}
   auth0SilentRenewUrl: document.baseURI + 'silentRenew',
   loginLandingUrl: document.baseURI + 'login-landing',
-  platformGAToken: "{{$conf.Data.platformGAToken}}",
-  projectGAToken:"{{$conf.Data.projectGAToken}}",
+  platformGAToken: "{{.Data.platformGAToken}}",
+  projectGAToken:"{{.Data.projectGAToken}}",
   doLocalRegistration:false,
-  mapsApiKey:"{{$conf.Data.mapsApiKey}}"
+  mapsApiKey:"{{.Data.mapsApiKey}}"
 };
 
-{{end}}{{end}}{{end}}
\ No newline at end of file
+{{end}}
```

Comment:
Converting ctmpl files to use more recent template syntax. Affects use of environment variables and of reading `secret`.